q3indel/test/au/edu/qimr/indel/IniFileTest.java

```diff
@@ -9,56 +9,66 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import junit.framework.Assert;
-
-import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
  
 
 public class IniFileTest {
-	public static final String ini = "test.ini";
-	public static final String output = "test.output.vcf";
-	public static final File fini = new File( ini);
 	
+	File fini;
+	File output;	
+	File blah;
 	
 	@org.junit.Rule
 	public  TemporaryFolder testFolder = new TemporaryFolder();
 	
-	@After
-	public void after() {	
-		fini.delete();
+	@Before
+	public void setup() throws IOException {
+		fini =  testFolder.newFile("test.ini");		
+		output = testFolder.newFile("test.output.vcf");
+		blah = testFolder.newFile("blah");	
+		
+		//must delete output the Option instance won't accept existing output
+		output.delete(); 
```

Comment:
I don't think you should need to create and then delete files. `TemporaryFolder` allows you to create a directory - maybe do that and have `output` and `blah` pointing to files (that don't yet exist) in the directory

Reply:
I tested it, it does create a real file. eg.   output = testFolder.newFile("output.vcf"); Assert.assertTrue(output.exists());

Reply:
but why are you then immediately deleting it?

Reply:
It only needs the file path. If the output file already exists, then q3indel will throw an exception.

Reply:
rather than creating and then deleting a file - why not create a directory for the output (as mentioned above)?

Reply:
OK, did and pushed